lab-jessica/lib/authentication.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const createError = require('http-errors');
+const User = require('../models/user.js');
+
+module.exports = (req, res, next) => {
+  const auth = req.headers.authorization;
+
+  if(!auth) throw new Error('authorization headers expected');
```

Comment:
I would suggest using a `createError()` here as well, which you can also pass through to `next()` as an argument.